docs/index.md

```diff
@@ -2,32 +2,43 @@
 
 <p style="font-size: larger; font-weight: bold;">Transform your research with vast amounts of computing!</p>
 
-The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC) systems&nbsp;&mdash;
-locally or using the OSG&nbsp;&mdash;
+The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC)&nbsp;&mdash;
+whether on a 'local' system or using the 
+[Open Science Grid (OSG)](https://www.youtube.com/watch?v=t2PzIy-vvaE&list=PLBWb4iScSWcPy7LQ4BuXmm8Z2xO4ZW1J_)&nbsp;&mdash;
 to run large-scale research simulation, analysis, and other applications for cutting-edge science.
-Through lecture, demonstration, hands-on exercises, and one-on-one consulting with experienced OSG staff,
+Through lecture, demonstration, hands-on exercises, and peronalized consulting with experienced OSG staff,
```

Comment:
Typo, and what’s the benefit here?

Reply:
I like emphasizing 'personalized' (thanks for the typo catch) and this gives us room to meet with more than one person at a time, if we need/want to.